src/workerd/api/pyodide/pyodide.c++

```diff
@@ -35,10 +35,9 @@ void PyodideBundleManager::setPyodideBundleData(
       kj::mv(version), {.messageReader = kj::mv(messageReader), .bundle = bundle});
 }
 
-const kj::Maybe<kj::ArrayPtr<const unsigned char>> PyodidePackageManager::getPyodidePackage(
+const kj::Maybe<const kj::Array<unsigned char>&> PyodidePackageManager::getPyodidePackage(
```

Comment:
What's the reason for this change?

Reply:
I was getting stack-use-after-scope errors before this change. I'm guessing it was returning a pointer to the reference, rather than the array itself. This seems to fix it.